feedback-module/src/components/Module.js

```diff
@@ -110,23 +112,34 @@ function Module({ pagetitle, endpoint, dir }) {
       pageTitleAsScreen(currentPageTitle);
       pageChange(currentPageTitle, nextPageTitle);
 
-      sendFormData(screen.formID),
-        setScreen(SCREENS[nextScreen]),
-        setCheckboxError(false);
+      const submissionObj = requestInfo(screen.formID);
+      if (submissionObj) {
+        setLoading(true);
+        requestService(
+          screen.formID,
+          submissionObj,
+          () => {
```

Comment:
Do you think it would look cleaner / be more readable if you defined this function outside of the requestService call?

Reply:
Yep, good idea! I think we actually reuse these 2 lines somewhere else too, so it would def be cleaner to make it into its own function!